nomos-services/data-availability/src/network/adapters/p2p.rs

```diff
@@ -7,7 +7,11 @@ use std::marker::PhantomData;
 // internal
 use crate::network::NetworkAdapter;
 use nomos_core::wire;
-use nomos_network::backends::libp2p::{Command, Event, EventKind, Libp2p, Message, TopicHash};
+#[cfg(feature = "libp2p")]
+use nomos_network::backends::libp2p::Libp2p as Backend;
+use nomos_network::backends::libp2p::{Command, Event, EventKind, Message, TopicHash};
```

Comment:
Are these also used by the mixnet implementation?

Reply:
Yes. Since the mixnet backend has the same API as the libp2p backend, these are also used by the mixnet backend.